compiler/circle-mpqsolver/src/pattern/PatternResolver.cpp

```diff
@@ -163,6 +163,105 @@ bool LayerNormPattern::matched()
   return true;
 }
 
+/**
+ * SUBGRAPH PATTERN
+ *  SoftmaxPattern := (exp(ifm - max(ifm))) / sum(exp(ifm - max(ifm)))
+ *    - Below diagram shows Softmax pattern to quantize.
+ *
+ *         [In]    [CircleConst] = [CircleConst(=-1)]
```

Comment:
It seems that this comment is misleading. Circle consts can be different objects.

Reply:
Ahh. Ok. Let it be `[CircleConst=1]   [CircleConst(=-1)]`.

Reply:
fixed